environments/conda-lock.yml

```diff
@@ -3934,52 +3894,52 @@ package:
     category: main
     optional: false
   - name: cryptography
-    version: 44.0.0
+    version: 43.0.3
```

Comment:
@bendnorman Looks like a rollback of the package that was causing problems - maybe this will fix #3985?

Reply:
Ah hopefully! I'll merge it in.